settings.gradle

```diff
@@ -5,5 +5,11 @@ include 'cristin-project'
 include 'cristin-organization'
 include 'cristin-person'
 include 'cristin-funding'
+include 'cristin-biobank'
 include 'cristin-testing'
+include 'cristin-biobank'
```

Comment:
I think you need only one entry with "include 'cristin-biobank'. Line 10-14 can be removed I think